src/pool.jl

```diff
@@ -189,7 +180,8 @@ function merge_pools(a::CategoricalPool{T}, b::CategoricalPool) where {T}
         newlevs = copy(levels(a))
         ordered = isordered(a)
     else
-        nl, ordered = mergelevels(isordered(a), a.levels, b.levels)
+        ordered = isordered(a) && (isordered(b) || b ⊆ a)
+        nl, ordered = mergelevels(ordered, a.levels, b.levels)
```

Comment:
Can you please remind me the rule when `b ⊆ a` is `true`?

Reply:
What do you mean? The idea is that if `a` is ordered and `b` a subset of `a`, there's no reason to care about whether `b` is ordered since `a` contains more information anyway.

Reply:
I wanted to make sure how `b ⊆ a` is defined. Is it:

> by removing some (or no) values of `a` without reordering the remaining values you can get `b`?

Reply:
No it's defined exactly like `issubset`, without taking order into account. But `mergelevels` only returns `ordered=true` if orders are compatible.

Reply:
Fair enough. Is this documented somewhere?

Reply:
Not really. I've added more docs (no docstring for `issubset` though since `CategoricalPool` isn't exported and I don't want to pollute the documentation).